### PR TITLE
Add test to verify IP is getting injected

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

File: tests/validation/cattlevalidationtest/core/test_container.py [hunk 1]
```diff
@@ -114,3 +115,36 @@ def test_linking(client, test_name, managed_network):
...
+    return logs, c, c.primaryIpAddress
+
+
+def test_ip_inject(admin_client, managed_network):
+    logs, c, ip = create_container(admin_client, managed_network)
+    conn = ws.create_connection(logs.url + '?token='+logs.token)
+
+    count = 0
+    found_ip = False
+    while True:
```
Comment (hibooboo2) [inline tests/validation/cattlevalidationtest/core/test_container.py]: Why not put the >100 condition in this while loop.

**Comment by hibooboo2** [comment]
LGTM

File: tests/validation/cattlevalidationtest/core/test_container.py [hunk 2]
```diff
@@ -114,3 +115,36 @@ def test_linking(client, test_name, managed_network):
...
+            result = conn.recv()
+            if ip in result:
+                found_ip = True
+                break
+            count += 1
+        except ws.WebSocketConnectionClosedException:
+            break
+
+    assert found_ip
+    c.stop(remove=True, timeout=0)
```
Comment (cloudnautique) [inline tests/validation/cattlevalidationtest/core/test_container.py]: If the assert fails it will leave the container. Can we make sure we clean up?

**Comment by cloudnautique** [comment]
LGTM

**Pull request merged**